tests/components/apple_tv/conftest.py

```diff
@@ -11,7 +12,7 @@
 
 
 @pytest.fixture(autouse=True, name="mock_scan")
-def mock_scan_fixture():
+def mock_scan_fixture() -> Generator[AsyncMock, None, None]:
```

Comment:
```suggestion
def mock_scan_fixture() -> Generator[AsyncMock]:
```
Same for the other instances.

Reply:
Adjusted